src/App.tsx

```diff
@@ -21,6 +21,7 @@ function App() {
 
   return (
     <div className="App">
+        <h2>Hallo</h2>
```

Comment:
```suggestion
        <h3>Hallo</h3>
```

use h3 insted, better for SEO

Reply:
Perfect!!!